apps/retrospective/services/sprint_task.go

```diff
@@ -311,3 +323,20 @@ func (service SprintTaskService) tasksWithTimeDetailsForCurrentAndPrevSprint(ret
 func (service SprintTaskService) AssignPointsToSprintTask(sprintTaskID string, sprintID string) {
 	workers.Enqueuer.EnqueueUnique("assign_points_to_sprint_task", work.Q{"sprintID": fmt.Sprint(sprintID), "sprintTaskID": fmt.Sprint(sprintTaskID)})
 }
+
+func RemoveDuplicatesFromSlice(list []string) string {
+	cleaned := []string{}
+	m := make(map[string]bool)
+
+	for _, item := range list {
```

Comment:
Please format the file properly, run `make fmt` to automatically format the files.

Reply:
Done.